packages/components/src/Combobox/components/ComboboxOption/ComboboxOption.pom.tsx

```diff
@@ -1,26 +1,37 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
-import { ComboboxOption as ComboboxOptionComponent } from "./ComboboxOption";
-import { ComboboxOption } from "../../Combobox.types";
-import { ComboboxContextProvider } from "../../ComboboxProvider";
+import noop from "lodash/noop";
+import { ComboboxOption } from "./ComboboxOption";
+import { ComboboxOptionProps } from "../../Combobox.types";
+import {
+  ComboboxContextProvider,
+  ComboboxProviderProps,
+} from "../../ComboboxProvider";
 
-export function renderOption(
-  id: string | number,
-  label: string,
-  selected: ComboboxOption[],
-  onSelect = jest.fn(),
-) {
+export function renderOption({
+  id,
+  label,
+  customRender,
+  selected,
+  onSelect = noop,
+}: {
+  id: string | number;
```

Comment:
I really like this small cleanup, I find it is much easier to pass an object with "known" keys instead of passing arguments and hoping I didn't confuse the order